### PR TITLE
Wire Redis for cache and sessions

### PR DESCRIPTION
**Pull request by tgalopin**

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -160,3 +162,12 @@ bazinga_geocoder:
             locale:  '%locale%'
             use_ssl: true
             api_key: '%env(GMAPS_API_KEY)%'
+
+snc_redis:
+    clients:
+        default:
+            type: predis
+            alias: default
+            dsn: "redis://%env(REDIS_HOST)%"
```
Comment (salahm) [inline app/config/config.yml]: Y U NO use default value ? (or update docker to handle this env var)

Comment (tgalopin) [inline app/config/config.yml]: What do you mean? I need this env var for staging (in Kubernetes, Redis is not local).

Comment (romain-pierre) [inline app/config/config.yml]: Good idea to set the host in a parameter! This also allow an easy switch between docker and local Redis (in dev).

**Comment by romain-pierre** [comment]
If you wanna use Redis for sessions management in all environments, you could also remove `var/sessions/.gitkeep`

**Comment by tgalopin** [comment]
I removed var/sessions and the line for it in docker-compose.yml.

**Pull request merged**